tests/test_examples.py

```diff
@@ -174,7 +174,7 @@ def method_inst(request):
     method._global.cache_clear()
 
 
-@pytest.mark.parametrize("method_inst", ["kalman", "trend_filtered"], indirect=True)
+@pytest.mark.parametrize("method_inst", ["kalman", "trend_filtered", "spectral"], indirect=True)
```

Comment:
There are other changes I need to make to the tests to get them to cover the Spectral class properly. We'll need to have test cases with cosine-spaced data for Chebyshev. It seems like right now there is a library of `funcs_and_derivs`, and they're all for equispaced points. If one of you has an idea how to extend the test code cleanly, I'd be interested.